packages/svelte/src/internal/client/reactivity/effects.js

```diff
@@ -237,11 +237,22 @@ export function destroy_effect(effect) {
 		remove(effect.dom);
 	}
 
+	var parent = effect.parent;
+
+	if (parent !== null && (parent.f & BRANCH_EFFECT) !== 0) {
```

Comment:
shouldn't it be this?

```suggestion
	if ((effect.f & BRANCH_EFFECT) !== 0) {
```